Tools/clinic/clinic.py

```diff
@@ -2415,7 +2414,7 @@ def __init__(
             *,
             filename: str,
             verify: bool = True,
-            limited_capi: bool = False,
+            limited_capi: bool
```

Comment:
It's kind of surprising for me to have a mandatory parameter after an optional parameter. Maybe move it before verify?